content/docs/installation/features.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Features
+description: cert-manager feature policy
+---
+
+New cert-manager features and functionality is often initially implemented behind a feature gate. This is so as to not break users with functionality that has not yet been tested in production as well as to give us a chance to remove or modify API fields and functionality following user feedback.
+
+We have alpha and beta features. We do not aim to keep any of the features in alpha or beta stage indefinitely. Feature gating should only be used for functionality that can eventually be turned on by default for all users (GA).
+
+A feature gate can be toggled on/off using `--feature-gates` flags on cert-manager controller. For feature gated functionality that comes with new API fields there is also a corresponding feature gate on webhook that also needs to be enabled using a `--feature-gates` flag if you want to use it.
+
+**Alpha**
+
+All alpha features are off by default.
+We keep the right to remove or change alpha features without a warning.
+An API field that is part of an alpha feature and requires a webhook feature flag to be used also can be removed from the API without a warning.
+An alpha feature might not work for all versions of Kubernetes supported by cert-manager.
+
+**Beta**
+
+All beta features are off by default.
+Beta features will not be removed, but can be changed.
+If the feature gets changed in incompatible ways, we will provide migration instructions.
+A beta feature will work with all versions of Kubernetes supported by cert-manager.
+
+**GA**
+
+A feature that is GA is on by default and cannot be disabled.
+With regards to API fields and their functionality, we keep Kubernetes API compatibility promise https://cert-manager.io/docs/installation/api-compatibility/.
+The feature flag for a GA feature will eventually be removed.
+
+## Graduation
+
+The graduation criteria can be different for each feature.
+Generally, we find user feedback most valuable when determining if a feature is sufficiently mature to graduate. If you are using an alpha or beta feature and would like to see it graduate, it would be great if you could give us some feedback about how you use it and whether you find the API useful to initiate graduation process. Feel free to [open a GitHub issue](https://github.com/cert-manager/cert-manager/issues/new/choose) or [join one of our meetings](https://cert-manager.io/docs/contributing/#meetings) to talk about this.
```

Comment:
nitpick: the meetings link is to the website, so it might be worth following the [linking to other pages](https://github.com/cert-manager/website/#task-linking-to-other-documentation-pages) guide.

Reply:
Thank for the reminder, that guide is great btw 👍🏼 I have changed all internal links.